components/bl-mapbox/styles/index.less

```diff
@@ -4,6 +4,7 @@
 
 @bl-customComponent-mapbox-height: 400px;
 @bl-customComponent-mapbox-width: 600px;
+@bl-customComponent-mapbox-marker-click-color: red;
```

Comment:
click-color -> active-color

And by default it should be the same as not active marker or 'unset'